Lib/profiling/sampling/__init__.py

```diff
@@ -7,6 +7,7 @@
 from .collector import Collector
 from .pstats_collector import PstatsCollector
 from .stack_collector import CollapsedStackCollector
+from .gecko_collector import GeckoCollector
 from .string_table import StringTable
 
-__all__ = ("Collector", "PstatsCollector", "CollapsedStackCollector", "StringTable")
+__all__ = ("Collector", "PstatsCollector", "CollapsedStackCollector", "GeckoCollector", "StringTable")
```

Comment:
ℹ️ **Maintainability**

Including `GeckoCollector` in `__all__` is a good practice as it explicitly defines the public API of the module. Ensure that `GeckoCollector` is well-documented and its interface is stable.